frontend/wallet/src/render/views/onboarding/AccountCreationStep.re

```diff
@@ -19,6 +19,12 @@ module Styles = {
     style([display(`flex), flexDirection(`row)]);
   };
 
+  let fadeIn =
```

Comment:
Might be worth making this part of theme? Would be even cooler if we could just wrap any component and have it animate the fade in.

```
<FadeIn duration={500} delay={200}>
  {children}
</FadeIn>
```